Annotations/AddAnnotation/AddAnnotation.csproj

```diff
@@ -21,7 +21,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
```

Comment:
The framework version is not proper. Check for all the projects

Reply:
Framework version check and modify all project is completed